roles/storage_nfs_lvm/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Create lvm volumes
+  lvol: vg={{volume_group}} lv={{ item }} size={{volume_size}}G
+  with_sequence: start={{volume_num_start}} count={{number_of_volumes}} format={{volume_prefix}}{{volume_size}}g%04d
+
+- name: create filesystem
+  filesystem: fstype=xfs dev=/dev/{{volume_group}}/{{ item }}
+  with_sequence: start={{volume_num_start}} count={{number_of_volumes}} format={{volume_prefix}}{{volume_size}}g%04d
+
+- name: mount volumes
+  mount: name={{mount_dir}}/{{ item }} src=/dev/{{volume_group}}/{{ item }} state=mounted fstype=xfs passno=0
+  with_sequence: start={{volume_num_start}} count={{number_of_volumes}} format={{volume_prefix}}{{volume_size}}g%04d
+
+- name: Make mounts owned by nfsnobody
+  command: "chown nfsnobody:nfsnobody {{mount_dir}}/{{ item }}"
+  with_sequence: start={{volume_num_start}} count={{number_of_volumes}} format={{volume_prefix}}{{volume_size}}g%04d
+
+- name: Make mounts nfsnobody only accessable
+  command: "chmod 700 {{mount_dir}}/{{ item }}"
```

Comment:
Why not use the [file module](http://docs.ansible.com/file_module.html) to do this chmod?